.github/workflows/src/spec-gen-sdk-status.js

```diff
@@ -16,10 +16,6 @@ import {
  * @returns {Promise<void>}
  */
 export default async function setSpecGenSdkStatus({ github, context, core }) {
-  const adoToken = process.env.ADO_TOKEN;
```

Comment:
Don't you still need to get the token?

Reply:
This line of code was for debugging. Here is [the code](https://github.com/Azure/azure-rest-api-specs/pull/34945/files#diff-410f1d9b7cf1ea63bd905c016d27c33f001af44ae4f67162acb8d68d0831391eR168) to get token.